int32 id
string name
float32 mean_error
geometry_msgs/Pose pose
geometry_msgs/Twist velocity



